styles/styles.css

```diff
@@ -156,48 +155,153 @@ a:hover {
 }
 
 /* buttons */
-a.button:any-link,
-button {
+.button {
+  --color-text-button-disabled: rgb(from var(--spectrum-black) r g b / 22%);
+
   box-sizing: border-box;
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 3rem;
   max-width: 100%;
-  margin: 0.75rem 0;
-  border: 2px solid transparent;
-  border-radius: var(--corner-radius-button);
-  padding: 0.5em 1.2em;
+  min-height: 3rem;
+  padding: 0.5rem 1.25rem;
+  border: none;
+  border-radius: 1.5rem;
+  background-color: transparent;
+  color: inherit;
   font-family: var(--body-font-family);
+  font-size: var(--spectrum-font-size-300);
   font-style: normal;
-  font-weight: var(--spectrum-medium-font-weight);
-  line-height: 1.25;
+  font-weight: var(--spectrum-bold-font-weight);
+  line-height: 1.3;
   text-align: center;
   text-decoration: none;
-  background-color: var(--color-background-button);
-  color: var(--color-text-button);
-  cursor: pointer;
   overflow: hidden;
-  text-overflow: ellipsis;
   white-space: nowrap;
+  text-overflow: ellipsis;
+  cursor: pointer;
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:disabled,
+  &:disabled:hover {
+    color: var(--color-text-button-disabled);
+    cursor: unset;
+  }
+
+  &:focus,
+  &:focus-visible {
+    outline: 2px solid var(--spectrum-focus-indicator-color);
+    outline-offset: 2px;
+  }
 }
 
-a.button:hover,
-a.button:focus,
-button:hover,
-button:focus {
-  background-color: var(--color-background-button-hover);
-  cursor: pointer;
+.button--primary {
+  --color-button-primary-background: rgb(from var(--spectrum-black) r g b / 84%);
+  --color-button-primary-background-hover: rgb(from var(--spectrum-black) r g b / 93%);
+  --color-button-primary-background-disabled: rgb(from var(--spectrum-black) r g b / 9%);
+  --color-text-button-primary: var(--spectrum-white);
+  
+  background-color: var(--color-button-primary-background);
+  color: var(--color-text-button-primary);
+
+  &:hover {
+    background-color: var(--color-button-primary-background-hover);
+    color: var(--color-text-button-primary);
+  }
+
+  &:disabled,
+  &:disabled:hover {
+    background-color: var(--color-button-primary-background-disabled);
+  }
 }
 
-button:disabled,
-button:disabled:hover {
-  opacity: var(--spectrum-opacity-disabled);
-  cursor: unset;
+:root:has(#color-scheme option[value="dark"]:checked) .button--primary {
+  --color-button-primary-background: rgb(from var(--spectrum-white) r g b / 85%);
+  --color-button-primary-background-hover: rgb(from var(--spectrum-white) r g b / 94%);
+  --color-button-primary-background-disabled: rgb(from var(--spectrum-white) r g b / 11%);
+  --color-text-button-primary: var(--spectrum-gray-25);
```

Comment:
i'm not going to make a million comments about cleaning up your variable names, I think you probably get it.